sycl/test-e2e/Plugin/sycl-ls-uuid-subdevs.cpp

```diff
@@ -0,0 +1,15 @@
+/* Test to check that sycl-ls is outputting UUID and number of sub and sub-sub
+ * devices. */
+// REQUIRES:  gpu, level_zero
```

Comment:
will this work on Gen12 ? Or is it PVC only?

Reply:
It should work on Gen12 as well. The test passes on Gen12 precommit: https://github.com/intel/llvm/actions/runs/9343215312/job/25714501971